core/primitives-core/src/runtime/fees.rs

```diff
@@ -136,8 +135,8 @@ pub struct StorageUsageConfig {
     pub num_extra_bytes_record: u64,
 }
 
-impl Default for RuntimeFeesConfig {
-    fn default() -> Self {
+impl RuntimeFeesConfig {
+    pub fn test() -> Self {
```

Comment:
Rather than hard-coding a bunch of constants here, can we re-define this as

```
fn test() {
   RuntimeStoreConfig::new(None).for_protocol_version(PROTOCL_VERSION)
}
```

?

Reply:
That's a good point, thank you!

I need to check if this doesn't break tests, and this PR is going to have some meaningful logic already, so I prefer to try it in the following PR if you don't mind.